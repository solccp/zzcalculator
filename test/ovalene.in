   32
34
 C     4.849981     0.000000     2.800137
 C     4.849981     0.000000     1.400068
 C     3.637486     0.000000     0.700034
 C     2.424991     0.000000     1.400069
 C     2.424990     0.000000     2.800138
 C     3.637485     0.000000     3.500172
 C     1.212495     0.000000     0.700034
 C     1.212495     0.000000    -0.700035
 C     2.424991     0.000000    -1.400069
 C     3.637486     0.000000    -0.700035
 C    -0.000001     0.000000     1.400069
 C    -1.212496     0.000000     0.700034
 C    -1.212495     0.000000    -0.700036
 C     0.000000     0.000000    -1.400069
 C     6.062475     0.000000     0.700034
 C     4.849981     0.000000    -1.400069
 C     0.000000     0.000000    -2.800137
 C     2.424992     0.000000    -2.800137
 C     6.062475     0.000000    -0.700034
 C     1.212497     0.000000    -3.500171
 C     4.849981     0.000000    -2.800137
 C     3.637486     0.000000    -3.500171
 C     1.212495     0.000000     3.500171
 C    -0.000001     0.000000     2.800137
 C     7.274969     0.000000    -1.400069
 C     1.212498     0.000000    -4.900239
 C     6.062475     0.000000    -3.500171
 C     3.637487     0.000000    -4.900239
 C     7.274968     0.000000    -2.800137
 C     2.424994     0.000000    -5.600272
 C     6.062475     0.000000    -4.900239
 C     4.849982     0.000000    -5.600272
