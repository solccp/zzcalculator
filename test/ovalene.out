 50 + 100 x + 66 x^2 + 16 x^3 + 1 x^4
 total: 233
